tests/MongoDB.Analyzer.Tests/Infrastructure/DiagnosticsAnalyzer.cs

```diff
@@ -46,20 +48,16 @@ internal static class DiagnosticsAnalyzer
 
         var allReferences = netReferences
             .AddPackages(packages)
-            .AddAssemblies(ImmutableArray.Create(PathUtilities.TestDataModelAssemblyPath))
+            .AddAssemblies(ImmutableArray.Create(testDataModelAssembly))
             .WithNuGetConfigFilePath(PathUtilities.NugetConfigPath);
 
         var metadataReferences = await allReferences.ResolveAsync(LanguageNames.CSharp, default);
 
         var testCodeText = File.ReadAllText(testCodeFilename);
         var testCodeSyntaxTree = CSharpSyntaxTree.ParseText(testCodeText);
-        var compilationOptions = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
 
-        var compilation = CSharpCompilation.Create(
-            "TestAssembly",
-            new[] { testCodeSyntaxTree },
-            metadataReferences,
-            compilationOptions);
+        var compilationOptions = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
```

Comment:
If there are no changes here, revert to keep the styling?

Reply:
Done